### DISCUSSION TITLE
api.mason-registry.dev is not accessible

### DISCUSSION THREAD

**Discussion by oizmerly** [mason-org/mason.nvim#827] [Q&A]
I use a system which cannot connect `https://api.mason-registry.dev` due to some network firewall limitations. Also with no root access it is impossible to add api.mason-registry.dev to /etc/hosts. Is there any way to overcome this problem?

**Comment by williamboman** [comment]
Hello! You should be able to switch the provider that's used, like so:

```lua
    require("mason").setup {
        providers = {
            "mason.providers.client",
        }
    }
```

The `mason.providers.client` provider will call service APIs directly to resolve package metadata.
